runtime/near-vm-runner/src/near_vm_runner.rs

```diff
@@ -9,53 +9,55 @@ use near_primitives::contract::ContractCode;
 use near_primitives::runtime::fees::RuntimeFeesConfig;
 use near_primitives::types::{CompiledContract, CompiledContractCache};
 use near_stable_hasher::StableHasher;
+use near_vm_compiler_singlepass::Singlepass;
+use near_vm_engine::{Engine, Executable};
+use near_vm_engine_universal::{
+    Universal, UniversalEngine, UniversalExecutable, UniversalExecutableRef,
+};
 use near_vm_errors::{
     CacheError, CompilationError, FunctionCallError, MethodResolveError, VMRunnerError, WasmTrap,
 };
 use near_vm_logic::gas_counter::FastGasCounter;
 use near_vm_logic::types::{PromiseResult, ProtocolVersion};
 use near_vm_logic::{External, MemSlice, MemoryLike, VMConfig, VMContext, VMLogic, VMOutcome};
+use near_vm_types::{Features, FunctionIndex, InstanceConfig, MemoryType, Pages, WASM_PAGE_SIZE};
+use near_vm_vm::{
+    Artifact, Instantiatable, LinearMemory, LinearTable, Memory, MemoryStyle, TrapCode, VMMemory,
+};
 use std::borrow::Cow;
 use std::hash::{Hash, Hasher};
 use std::mem::size_of;
 use std::sync::Arc;
-use wasmer_compiler_singlepass::Singlepass;
-use wasmer_engine::{Engine, Executable};
-use wasmer_engine_universal::{
-    Universal, UniversalEngine, UniversalExecutable, UniversalExecutableRef,
-};
-use wasmer_types::{Features, FunctionIndex, InstanceConfig, MemoryType, Pages, WASM_PAGE_SIZE};
-use wasmer_vm::{
-    Artifact, Instantiatable, LinearMemory, LinearTable, Memory, MemoryStyle, TrapCode, VMMemory,
-};
 
-const WASMER_FEATURES: Features = Features {
+const VM_FEATURES: Features = Features {
     threads: WASM_FEATURES.threads,
     reference_types: WASM_FEATURES.reference_types,
     simd: WASM_FEATURES.simd,
     bulk_memory: WASM_FEATURES.bulk_memory,
     multi_value: WASM_FEATURES.multi_value,
     tail_call: WASM_FEATURES.tail_call,
-    module_linking: WASM_FEATURES.module_linking,
     multi_memory: WASM_FEATURES.multi_memory,
     memory64: WASM_FEATURES.memory64,
     exceptions: WASM_FEATURES.exceptions,
+    mutable_global: true,
+    saturating_float_to_int: true,
+    sign_extension: true,
```

Comment:
We went over this before, but double-checking these ideally happens before we roll something out (you mentioned believing these should be true, I still haven't double checked it, though.)

Reply:
This message seems to confirm that these were on-by-default in previous versions of wasmparser: https://github.com/bytecodealliance/wasm-tools/pull/482